src/components/Search/Search.js

```diff
@@ -83,13 +93,15 @@ function Search({
             items ? items.map(i => ({ ...i, section })) : []
           }
           getSuggestionValue={suggestion => searchService.value(suggestion)}
-          onSuggestionsFetchRequested={() => searchService.search(value)}
+          onSuggestionsFetchRequested={suggestion =>
```

Comment:
Good catch! How about:
```suggestion
          onSuggestionsFetchRequested={({value: newValue}) =>
```
Because function param is not really the suggestion.